.github/workflows/pull-request.yml

```diff
@@ -21,15 +21,15 @@ jobs:
   prettier:
     name: prettier has been run on compatible files
     runs-on: ubuntu-latest
-    container: node:14
+    container: node:18
```

Comment:
I _think_ we could replace this container selection with the more generic setup-node action, for prettier and eslint, and not have to touch them again when we move to node20

```yaml
      - uses: actions/setup-node@v3
        with:
          node-version-file: ".nvmrc"
```

Reply:
good callout. I'll go ahead and make that swap